lxd/db/openfga/openfga.go

```diff
@@ -231,6 +330,23 @@ func (o *openfgaStore) ReadUsersetTuples(ctx context.Context, store string, filt
 		return nil, fmt.Errorf("ReadUsersetTuples: Failed to parse entity URL %q: %w", entityURL, err)
 	}
 
+	// Check the request cache.
+	cache, err := request.GetCtxValue[*OpenFGACache](ctx, request.CtxOpenFGARequestCache)
+	if err != nil {
+		return nil, fmt.Errorf("ReadUsersetTuples: Failed to get OpenFGA request ID from context: %w", err)
```

Comment:
I don't think we should error out if the cache is not present. We may not always want to use the cache. For example on long-running connections like `/1.0/events`. Instead I think we should just call the database as normal.

Reply:
yes ok. fixed